<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParleVision</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="topContainer">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="plvgui::PipelineView" name="view">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="utilityContainer" stretch="0">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Maximum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>240</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menuActions">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_Library"/>
    <addaction name="actionShow_Inspector"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuActions"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShow_Library"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionLoad"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionShow_Inspector"/>
  </widget>
  <action name="actionStart">
   <property name="icon">
    <iconset>
     <normalon>:/icons/play.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="iconText">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start the pipeline</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset>
     <normalon>:/icons/stop.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="iconText">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop the pipeline</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset>
     <normalon>:/icons/pause.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normalon>:/icons/save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save the active pipeline to a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset>
     <normalon>:/icons/folder.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="iconText">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load a pipeline from a file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionShow_Library">
   <property name="icon">
    <iconset>
     <normalon>:/icons/library.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Show Library</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionShow_Inspector">
   <property name="icon">
    <iconset>
     <normalon>:/icons/inspector.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Show Inspector</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>plvgui::PipelineView</class>
   <extends>QGraphicsView</extends>
   <header>plvgui/pipelineview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
